utils/lru_test.go

```diff
@@ -20,49 +21,42 @@ func TestLRU(t *testing.T) {
 	for i := 0; i < 256; i++ {
 		l.Add(i, i)
 	}
-	if l.Len() != 128 {
-		t.Fatalf("bad len: %v", l.Len())
-	}
+	require.Equalf(t, 128, l.Len(), "bad len: %v", l.Len())
 
 	for i, k := range l.Keys() {
-		if v, ok := l.Get(k); !ok || v != k || v != i+128 {
-			t.Fatalf("bad key: %v", k)
-		}
+		v, ok := l.Get(k)
+		require.Equalf(t, true, ok, "bad key: %v", k)
```

Comment:
Could we use `require.True` to be more semantic here?

Reply:
sure